src/styles/blocks/calendar.scss

```
@@ -0,0 +1,89 @@
+
+.calendar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__days {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-size;
+    margin: $blocks-padding;
+
+    @for $day from 1 through 7 {
+      &--#{$day} {
+        box-sizing: border-box;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        background-color: rgb(76, 231, 218);
+        height: 45px;
+        width: $squre-size;
+        border: 1px solid black;
+        margin: $small-blocks--margin;
```

а чому не можна це задати для їх загального селектору?
```suggestion
    @for $day from 1 through 7 {
      &--#{$day} {
        box-sizing: border-box;
        display: flex;
        justify-content: center;
        align-items: center;
        background-color: rgb(76, 231, 218);
        height: 45px;
        width: $squre-size;
        border: 1px solid black;
        margin: $small-blocks--margin;
```